cmd/core-data/res/configuration.yaml

```diff
@@ -9,16 +9,22 @@ Writable:
 #    Tags: # Contains the service level tags to be attached to all the service's metrics
     ##    Gateway="my-iot-gateway" # Tag must be added here or via Consul Env Override can only change existing value, not added new ones.
 Service:
-  Port: 59880
+  Port: 59810
   Host: "localhost"
   StartupMsg: "This is the Core Data Microservice"
+  SecurityOptions:
+    Mode: "zerotrust"
+    OpenZitiController: "openziti:1280"
+    OpenZitiServiceName: "edgex.core-data"
 
 MessageBus:
   Optional:
     ClientId: "core-data"
 
 Database:
   Name: "coredata"
+  Host: "127.0.0.1"
+  Port: 6379
```

Comment:
This is already set in common config. 
When running from command line in hybrid mode use `-cp -d -o` flags